webapp/channels/src/packages/mattermost-redux/src/selectors/entities/preferences.ts

```diff
@@ -245,6 +245,14 @@ export function isCustomGroupsEnabled(state: GlobalState): boolean {
     return getConfig(state).EnableCustomGroups === 'true';
 }
 
+export function getIsOnboardingFlowEnabled(state: GlobalState): boolean {
+    const isCloud = getLicense(state)?.Cloud === 'true';
+    if (isCloud) {
+        return true;
+    }
```

Comment:
Do we need this check for cloud? For Cloud we have the `EnableOnboardigFlow` disabled - not sure if any RFQA servers are running a cloud license